### PR TITLE
[IMP] payment: add is_post_processed field in the view

### PR DESCRIPTION
**Pull request by arj-odoo**
task id: 3258616

Description of the issue/feature this PR addresses:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/payment/views/payment_transaction_views.xml [hunk 1]
```diff
@@ -38,6 +38,7 @@
                             <field name="token_id" attrs="{'invisible': [('token_id', '=', False)]}"/>
                             <field name="create_date"/>
                             <field name="last_state_change"/>
+                            <field name="is_post_processed"/>
```
Comment (AntoineVDV) [inline addons/payment/views/payment_transaction_views.xml:41]:
```suggestion
                            <field name="is_post_processed" groups="base.group_no_one"/>
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
This view is also accessible without debug mode.

**Review by AntoineVDV — APPROVED** [review]

**Comment by arj-odoo** [comment]
@robodoo r+

**Comment by arj-odoo** [comment]
@robodoo r+

**Pull request closed without merging**